src/api/app/models/comment.rb

```diff
@@ -13,8 +13,8 @@ class Comment < ApplicationRecord
 
   validate :validate_parent_id
 
-  after_create :create_event
-  after_destroy :delete_parent_if_unused
+  after_create :create_event, :instrument_diff_comments_on_create
+  after_destroy :delete_parent_if_unused, :instrument_diff_comments_on_destroy
```

Comment:
Why distinguish by state?

Reply:
To get the total number of times the feature has been used. `diff_comment` values are different for both states `(add/remove)`.